game_api.txt

```diff
@@ -64,7 +64,7 @@ Beds API
 Bones API
 ---------
 
-The string setting `on_die` will be read from the minetest.conf file ("bones" by default).
+The setting `bones_mode` is read from the minetest.conf file `bones_mode = bones` by default.
```

Comment:
Should be
```bones_mode = "bones"```
quotes are needed.
This is a setting not an API so this should be in minetest.conf_example? with the same format: Explanation first then the default setting on a newline after.